packages/contracts/contracts/generator/scripty-support/ETHFSFileStorage.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+///////////////////////////////////////////////////////////
+// ░██████╗░█████╗░██████╗░██╗██████╗░████████╗██╗░░░██╗ //
+// ██╔════╝██╔══██╗██╔══██╗██║██╔══██╗╚══██╔══╝╚██╗░██╔╝ //
+// ╚█████╗░██║░░╚═╝██████╔╝██║██████╔╝░░░██║░░░░╚████╔╝░ //
+// ░╚═══██╗██║░░██╗██╔══██╗██║██╔═══╝░░░░██║░░░░░╚██╔╝░░ //
+// ██████╔╝╚█████╔╝██║░░██║██║██║░░░░░░░░██║░░░░░░██║░░░ //
+// ╚═════╝░░╚════╝░╚═╝░░╚═╝╚═╝╚═╝░░░░░░░░╚═╝░░░░░░╚═╝░░░ //
+///////////////////////////////////////////////////////////
+
+import {IFileStore} from "./ethfs/IFileStore.sol";
+import {IContractScript} from "scripty.sol/contracts/scripty/interfaces/IContractScript.sol";
+
+contract ETHFSFileStorage is IContractScript {
+    IFileStore public immutable fileStore;
+
+    constructor(address _fileStoreAddress) {
+        fileStore = IFileStore(_fileStoreAddress);
+    }
+
+    // =============================================================
+    //                            GETTERS
+    // =============================================================
+
+    /**
+     * @notice Get the full script from ethfs's FileStore contract
+     * @param name - Name given to the script. Eg: threejs.min.js_r148
+     * @param data - Arbitrary data. Not used by this contract.
+     * @return script - Full script from merged chunks
+     */
+    function getScript(
+        string calldata name,
+        bytes memory data
```

Comment:
unused function parameter - throws compiler warning, could comment out the name like this + update natspec
```suggestion
        bytes memory /*data*/
```